README.md

```diff
@@ -39,15 +39,15 @@ A youtube gameplay video can be found here: https://www.youtube.com/watch?v=-PEK
 * Changes in UI
   * We have added an instructions scene and a stunning menu with automatic gameplay on the background
 * Changes in logic
-  * We have fixed uncountable bugs which made the game unplayable
+  * We have fixed a very countable number of bugs
 
 <br></br>
-In short, we have turned a quite buggy game into a good looking, bug-free and addictive piece of art.
+In short, we have turned a quite buggy game into a good looking, bug-free and addictive piece of art while destroying the core game design that made Chromacore interesting.  The original authors released an updated version post-GitHub Game Off, [here](https://github.com/Murkantilism/Chromacore), preserving what made the game fun to begin with.
```

Comment:
For both nostalgia's sake and just to affirm I wasn't remembering things wrong just because my subconscious wanted me to remember things positively, I downloaded the build from the [competition repository](https://github.com/Murkantilism/game-off-2013) to replay it. Seems [pretty playable to me](https://youtu.be/UzprJvu3uEI).

I ran into bugs sure, sometimes they killed me and I had to respawn, but never was I unable to continue play. The bug we see in that video is due to animations and a lazy way of checking for death. On collision with a box it _was_ checking the animator's current animation state, so if you grabbed a lum right before the box you'd be in the "wrong" animation state and die. Notice on the second death in the video I purposefully use the punch ability to show that bug.

I have 2 guesses for why you chose to call it unplayable:

1. You tried the game for 2 mins, hit that bug, gave up, and dubbed it unplayable. If that's the criteria for an unplayable game then ~70-80% of AAA games released in the past 10 years are/were unplayable upon their release. Bit of a stretch if you ask me.

2. Or it's possible the game was "literally unplayable", if you tried working with the 2013 repository with a newer version of the Unity API. Notice in the video there are compiler errors that prevent play; these came up immediately after Unity's automated attempt at upgrading the project to the latest version ( that process is not shown in the video cause it took forever). It took me maybe 60 seconds to resolve those errors, so if this is the cause for your "unplayable" description, I'd argue that's not how that works. It'd be perfectly playable if you downloaded and installed the exact version of Unity the 2013 repository was built with; what made it unplayable was software on your end and frustration that you had to do some amount of work to get it working with whatever the latest version was at the time.